packaging/installer/functions.sh

```diff
@@ -303,7 +303,7 @@ install_non_systemd_init() {
 				run rc-update add netdata default &&
 				return 0
 
-		elif [ "${key}" = "debian-7" ] || [ "${key}" = "ubuntu-12.04" ] || [ "${key}" = "ubuntu-14.04" ]; then
+		elif [[ ${key} =~ (debian-[789]).* ]] || [ "${key}" = "ubuntu-12.04" ] || [ "${key}" = "ubuntu-14.04" ]; then
```

Comment:
debian 8 and 9 are init based, not systemd?

Reply:
hm, i fall in the dockerized version trap again? 
I did not find anything in the release notes of both versions either
Let me check again

Reply:
What's wrong with me? i did not find it in the first read through
https://www.debian.org/releases/jessie/amd64/release-notes/ch-whats-new.en.html#systemd

I was also a bit misguided from a bug report on a devel version of jessie that fall into that branch. 
I 'll use a droplet to test again